src/PlusDataCollection/WinProbe/vtkPlusWinProbeVideoSource.cxx

```diff
@@ -222,7 +222,7 @@ int vtkPlusWinProbeVideoSource::MSecondsFromWidth(int32_t value)
 
 // ----------------------------------------------------------------------------
 vtkPlusWinProbeVideoSource* thisPtr = nullptr;
-int32_t quadBFCount = 0; // number of 4x beamformers
+int32_t quadBFCount = 1; // number of 4x beamformers
```

Comment:
Comment could improve here as a value of 1 doesn't refer to 1 count of 4x beamformers based on how you use this variable.

Reply:
The use of this variable looks correct to me.

Reply:
The usage of the variable is correct. The comment is not as it makes it read as "1 number of 4x beamformers". Really it is like a True/False state for 8x/4x respectively so a change is needed to imrpove the specification that this specification of 1 refers to an 8x beamformer.

Reply:
the variable doesnt directly correspond to the ::getARFIIsx8Enabled() return, it's set to either 1 or 2 depending on that parameter, 2 for x8, so i think the comment makes sense here

Reply:
Ok sounds good